platform/android/CHANGELOG.md

```diff
@@ -11,6 +11,20 @@ Mapbox welcomes participation and contributions from everyone.  If you'd like to
 
  - Implemented asymmetric center of perspective: fixed an issue that caused the focal point to be always based on the view's horizontal center when setting [MapboxMap setPadding](https://docs.mapbox.com/android/api/map-sdk/8.0.0/com/mapbox/mapboxsdk/maps/MapboxMap.html#setPadding-int-int-int-int-). [#14664](https://github.com/mapbox/mapbox-gl-native/pull/14664)
 
+## 8.1.0 - June 20, 2019
+
+### Bugs
+- Accept floating numbers during core->platform color conversion [#14954](https://github.com/mapbox/mapbox-gl-native/pull/14954)
+- Added CameraMode.NONE_COMPASS and CameraMode.NONE_GPS to LocationModesActivity [#14945](https://github.com/mapbox/mapbox-gl-native/pull/14945)
```

Comment:
This is only a test app change, let's skip it.

Reply:
Removed